generate_index.py

```
@@ -0,0 +1,197 @@
+                    category = "Uncategorized"
+
+                if category.startswith('.'): # Skip hidden folders at root
+                     continue
+
+                title = get_html_title(filepath)
+                structure[category].append((title, rel_path))
+
+    # Sort categories and files
+    sorted_categories = sorted(structure.keys())
+
+    current_time = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Ruff (0.15.1)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)